docs/content/python/getting-started/quickstart/index.md

```diff
@@ -42,22 +40,24 @@ role-based access control.
 
 {{% exampleGet "import_code" %}}
 
+To refer to your {{% exampleGet "classes" %}} in Polar, you must _register_ them with Oso.
+
+{{% exampleGet "register_classes" %}}
+
 ## Accept or deny requests
 
-When a request arrives, your application will need to ask Oso if it should accept the request. Oso needs three pieces of information to make that decision:
+Oso needs three pieces of information to make an authorization decision:
 - Who is making the request? (the "actor")
 - What are they trying to do? (the "action")
 - What are they doing it to? (the "resource")
 
 You'll pass these pieces of information are to Oso's `{{% exampleGet "isAllowed" %}}` method: `{{% exampleGet "isAllowed" %}}(actor, action, resource)`.
 `{{% exampleGet "isAllowed" %}}` will return `True` or `False`, and your application can choose how to enforce that decision.
 
-That enforcement can happen in the request handler, at the database layer, or in middleware — here, we've chosen to enforce in the request handler.
-Here's {{% exampleGet "example_app" %}} route that only displays a page if the current user is allowed to read it.
+Here's a program that only allows access to a page if the current user has a role that is allowed to read it.
```

Comment:
This isn't strictly correct.

The program only allows access to a page if the policy allows the current user to perform the read action on it.

Since the policy uses roles, the user must have a role that's allowed to read it.

How important do you think this distinction is?

Reply:
I was trying to summarize, but I think being precise is more important here. I'll be specific.